Dockerfile.migrations

```diff
@@ -1,4 +1,5 @@
 FROM node:16-alpine3.16 as base
+RUN apk add g++ make py3-pip
```

Comment:
@batebobo Let's add a comment on why we need these packages. If we need them how does it work without them now?

Please sync your branch with master as the node version has been bumped from 16 to 18 there.

Reply:
I forgot to ask a question on Discord about this. Without these I get an error that I need a python installation for `node-gyp` when running `docker-compose up`. I'll post a screenshot once I have a reliable internet connection to run a new `docker-compose up`

Reply:
<img width="1579" alt="Screenshot 2022-12-03 at 1 34 40 PM" src="https://user-images.githubusercontent.com/6130708/205438939-d43d4e61-eef2-4bfb-aa9c-4878c728cadf.png">
This ^

I'll try updating the node version and see if it helps 😄

Reply:
It somehow works now 😩 Thank you!